util/types/schemas/extensions.mol

```diff
@@ -293,93 +293,107 @@ table VerifiableHeader {
     header:                 Header,
     uncles_hash:            Byte32,
     extension:              BytesOpt,
+    parent_chain_root:      HeaderDigest,
 }
 
-table VerifiableHeaderWithChainRoot {
-    header:                 Header,
-    uncles_hash:            Byte32,
-    extension:              BytesOpt,
-    chain_root:             HeaderDigest,
-}
+vector VerifiableHeaderVec <VerifiableHeader>;
 
-vector VerifiableHeaderWithChainRootVec <VerifiableHeaderWithChainRoot>;
+vector FilteredBlockVec <FilteredBlock>;
 
 union LightClientMessage {
+    // A client asks the server for the last state of the chain.
     GetLastState,
     SendLastState,
-    GetBlockSamples,
-    SendBlockSamples,
-    GetBlockProof,
-    SendBlockProof,
-    GetTransactions,
-    SendTransactions,
+    // A client asks the server for the proof of the last state which the
+    // client known.
+    GetLastStateProof,
+    SendLastStateProof,
+    // A client asks the server for the proof of some blocks.
+    GetBlocksProof,
+    SendBlocksProof,
+    // A client asks the server for the proof of some transactions.
+    GetTransactionsProof,
+    SendTransactionsProof,
```

Comment:
```suggestion
    GetBlockProofs,
    SendBlockProofs,
    // A client asks the server for the proof of some transactions.
    GetTransactionProofs,
    SendTransactionProofs,
```

Reply:
Only one proof is returned, it proves all items.